docs/architecture-overview.md

```diff
@@ -20,11 +24,11 @@ On a high level, the application consists of two main parts, as shown below.
 
 ![Overview Diagram](./images/v4.png)
 
-1. CWA Backend: Handles submission and aggregation/distribution of diagnosis keys and configuration files.
+1. CWA Server: Handles submission and aggregation/distribution of diagnosis keys and configuration files.
 2. Verification Backend: Deals with test result verification and issues TANs. This backend is managed and deployed
```

Comment:
Maybe also change to 'Verification Server' to be consistent? (here and all other instances in this doc)